addons/website_blog/static/tests/tours/blog_manager_tour.js

```diff
@@ -0,0 +1,83 @@
+import { clickOnSave, registerWebsitePreviewTour } from "@website/js/tours/tour_utils";
+import { _t } from "@web/core/l10n/translation";
```

Comment:
As it is test tour, translation is not needed.

Reply:
Thank you for the clarification!
We will proceed accordingly.